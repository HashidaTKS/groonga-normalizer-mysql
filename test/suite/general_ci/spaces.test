register normalizers/mysql

normalize NormalizerMySQLGeneralCI " \tã€€"
