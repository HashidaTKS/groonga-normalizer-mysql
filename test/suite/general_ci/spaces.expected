register normalizers/mysql
[[0,0.0,0.0],true]
normalize NormalizerMySQLGeneralCI " \t　"
[[0,0.0,0.0],{"normalized":" \t　","types":[],"checks":[]}]
